LibAddonMenu-2.0/controls/dropdown.lua

```diff
@@ -423,7 +445,7 @@ function LAMCreateControl.dropdown(parent, dropdownData, controlName)
         local sortInfo = GrabSortingInfo(dropdownData.sort)
         control.m_sortType, control.m_sortOrder = SORT_TYPES[sortInfo[1]], SORT_ORDERS[sortInfo[2]]
     elseif dropdownData.choicesValues then
-        control.m_sortType, control.m_sortOrder = ZO_SORT_ORDER_UP, SORT_BY_VALUE
+        control.m_sortType, control.m_sortOrder = SORT_BY_VALUE, ZO_SORT_ORDER_UP
```

Comment:
This is the biggest "functional" change in the PR - I feel pretty confident that it was a bug and this was the proper fix, but I can't promise the behaviour won't change.